dotcom-rendering/src/web/components/Callout/Select.tsx

```diff
@@ -1,36 +1,43 @@
-import { CampaignFieldSelect } from '../../../types/content';
+import { Select as SourceSelect } from '@guardian/source-react-components';
+import type { CampaignFieldSelect } from 'src/types/content';
 import { FieldLabel } from './FieldLabel';
 
 type Props = {
+	validationErrors?: { [key in string]: string };
 	formField: CampaignFieldSelect;
 	formData: { [key in string]: any };
 	setFormData: React.Dispatch<React.SetStateAction<{ [x: string]: any }>>;
 };
 
-export const Select = ({ formField, formData, setFormData }: Props) => (
+export const Select = ({
+	validationErrors,
+	formField,
+	formData,
+	setFormData,
+}: Props) => (
 	<>
 		<FieldLabel formField={formField} />
-		<select
+		<SourceSelect
 			data-testid={`form-field-${formField.id}`}
-			required={formField.required}
-			value={
-				formField.id && formField.id in formData
-					? formData[formField.id]
-					: ''
-			}
+			error={validationErrors?.[formField.id]}
+			label={''}
+			value={formField.id in formData ? formData[formField.id] : ''}
 			onChange={(e) =>
 				setFormData({
 					...formData,
 					[formField.id]: e.target.value,
 				})
 			}
-		>
-			{formField.options &&
-				formField.options.map((option, index) => (
+			optional={!formField.required}
+			children={[
+				{ label: 'default', value: 'Please choose an option ...' },
+			]
+				.concat(formField.options)
```

Comment:
Source behaves in such a way that if a select box is mandatory, and the user has NOT changed the value, it will raise an error with the user. Therefore, we add a default 'Please choose an option' field to the start of any array of options passed down to us.